page_analyzer/templates/index.html

```diff
@@ -16,22 +16,32 @@
         <a class="navbar-brand" href="/">Анализатор страниц</a>
         <div id="navbarNav">
           <ul class="navbar-nav">
-            <li class="nav-item"><a class="nav-link" href="">Сайты</a></li>
+            <li class="nav-item">
+              <a class="nav-link" href="{{ url_for('get_urls') }}">Сайты</a>
+            </li>
           </ul>
         </div>
       </nav>
+      {% if message %}
```

Comment:
should be done [differently](https://flask.palletsprojects.com/en/2.2.x/patterns/flashing/)